docker-compose.yml

```diff
@@ -169,3 +169,72 @@ services:
       - ALLOW_EMPTY_PASSWORD=yes
     ports:
       - 6379:6379
+
+  database:
```

Comment:
This service has another `pg_sql` run in this docker-compose, please remove it

Reply:
@bochengyang yes I used old version not rebase yet. I fixed